src/data/data-table/shared/DataTable.js

```diff
@@ -42,6 +30,19 @@ import * as StateHistory from 'src/libs/state-history';
 import * as Style from 'src/libs/style';
 import * as Utils from 'src/libs/utils';
 
+import { concatenateAttributeNames } from '../entity-service/attribute-utils';
```

Comment:
I know it's unconventional in Terra UI to use relative imports outside the directory, but separating these helps highlight where components need to be updated to remove the assumption of using Entity Service.

Reply:
This comment might deserve to be in the code itself to explain :)

But just so I understand...is the idea to use a relative import to make an easy to grep string to help us keep track of (and maybe ultimately eliminate) this dependency?

Reply:
Added a comment in https://github.com/DataBiosphere/terra-ui/pull/4178/commits/d113cb121ca482d44c7a8f0736089d01e0baffc8.